src/interpreter.js

```diff
@@ -40,26 +40,27 @@ import {
 // I know, I know... But this is not meant to be nice or final code. Just a quick rough approach to get a better feeling on the AST shape.
 
 const compileSentence = (sentence) => {
-  if(sentence.nodeType === 'VariableDeclaration') {
-    const { variable, writeable, value } = sentence
-    return ` ${writeable ? 'let' : 'const'} ${variable.name} = ${compileExpression(value)} ;`
-  }
-  if(sentence.nodeType === 'Assignment') {
-    const { variable, value } = sentence
-    return ` ${variable.name} = ${compileExpression(value)} ;`
+  switch (sentence.nodeType) {
```

Comment:
oh, ok I didn't realize the scoping in switch sucked :(

Anyway I didn't get any error either from babel or runtime.

Regarding cases with braces and returns it is quite common for example in redux reducers :)